base/linalg/symmetric.jl

```diff
@@ -57,9 +57,9 @@ ctranspose(A::Hermitian) = A
 trace(A::Hermitian) = real(trace(A.data))
 
 #tril/triu
-tril(A::Hermitian,k::Integer=0) = tril(A.data,k)
+tril(A::Hermitian,k::Integer=0) = triu(A.data,-k)'
```

Comment:
and these versions don't respect the symmetric parts if you ask for triu including some of the subdiagonals or tril including some of the superdiagonals - needs a band-partial version of something like `LinAlg.copytri!` I think, if it were in-place